src/lib/virtio/net_driver.lua

```diff
@@ -52,7 +51,6 @@ function VirtioNetDriver:new(args)
 
    if args.use_checksum then
       self.transmit = self._transmit_checksum
-      self.want_features = self.want_features + C.VIRTIO_NET_F_CSUM
```

Comment:
If `use_checksum` needs `C.VIRTIO_NET_F_CSUM` and `C.VIRTIO_NET_F_CSUM` is now a minimum feature. Shouldn't transmit always be `transmit_checksum`, or C.VIRTIO_NET_F_CSUM shouldn't be in the minimum feature set?